src/en/clowncorps/src/eu/kanade/tachiyomi/extension/en/clowncorps/ClownCorps.kt

```diff
@@ -0,0 +1,175 @@
+package eu.kanade.tachiyomi.extension.en.clowncorps
+
+import android.app.Application
+import android.content.SharedPreferences
+import androidx.preference.PreferenceScreen
+import androidx.preference.SwitchPreferenceCompat
+import eu.kanade.tachiyomi.lib.textinterceptor.TextInterceptor
+import eu.kanade.tachiyomi.lib.textinterceptor.TextInterceptorHelper
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.ConfigurableSource
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Response
+import rx.Observable
+import uy.kohesive.injekt.Injekt
+import uy.kohesive.injekt.api.get
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ClownCorps : ConfigurableSource, HttpSource() {
+    override val baseUrl = "https://clowncorps.net"
+    override val lang = "en"
+    override val name = "Clown Corps"
+    override val supportsLatest = false
+
+    private val creator = "Joe Chouinard"
+
+    // Text from: https://clowncorps.net/about/
+    private val synopsis = "Clown Corps is a comic about crime-fighting clowns. " +
+        "It's pronounced \"core.\" Like marine corps."
```

Comment:
Inline it.

Reply:
Done in 576a230f2a118b2bf12fae4fbf1256974af08105